collector/cluster_health.go

```diff
@@ -217,21 +237,31 @@ func (c *ClusterHealth) fetchAndDecodeClusterHealth() (clusterHealthResponse, er
 	}
 
 	if err := json.NewDecoder(res.Body).Decode(&chr); err != nil {
+		c.jsonParseFailures.Inc()
 		return chr, err
 	}
 
 	return chr, nil
 }
 
 func (c *ClusterHealth) Collect(ch chan<- prometheus.Metric) {
+	c.totalScrapes.Inc()
+	defer func() {
+		ch <- c.up
+		ch <- c.totalScrapes
+		ch <- c.jsonParseFailures
+	}()
+
 	clusterHealthResponse, err := c.fetchAndDecodeClusterHealth()
 	if err != nil {
+		c.up.Set(0)
```

Comment:
For certain definitions of correct ;)
It really depends how you define `up` in this case. But I'm completely open to suggestions. How should we define `up`? I used it as `the cluster is reachable via HTTP and answering with an HTTP Status 200`. Seems to be the same definition used by the HAProxy exporter. For the case that JSON decoding fails we'd still have the other metric.

But we can change the definition of up, of course. As longs as it's more or less consistent with other exporters.